api/views/CloudRecordSummaryView.py

```diff
@@ -174,6 +141,7 @@ def get(self, request, format=None):
 
     def _parse_query_parameters(self, request):
         """Parse expected query parameters from the given HTTP request."""
+        logger = logging.getLogger(__name__)
```

Comment:
Does having the `getLogger` duplicated here and below get you the method names for the logger name?